README.md

```diff
@@ -1 +1,38 @@
 # hw01
+## 10/04/2020
+***
+
+Hi everyone, my name is Emily Robb and I'm a fourth year in the College. I'm double-majoring in Philosophy and Political Science and am incredibly excited to take this course and learn practical skills that I haven't really had previous exposure to. Looking forward to learning with all of you!
+
+![Picture of me](IMG_5373.jpeg)
+
+---
+
+##### Process Description
+I started this homework by reviewing the overview and instructions, and ultimately watching the accompanying video.
+
+[Panopto Instructions](https://uchicago.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=7bd284fa-ceac-400b-a8db-ab9500f8d943&start=199.610526)
+
+I clicked the **code** button at the top of the page and forked my own copy of the hw01 file in order to edit it. 
+
+Next, I cloned my fork to my own computer. This process has a couple of steps that I took.
+
+1. Clicked the green clone button
+
+2. Copied the link to my clipboard
+
+3. Opened RStudio and created a new *version control* project using the repository URL copied in the last step
+
+To finish the rest of this homework up, I clicked on the README.md file and started writing in it with Markdown, adding different stylistic elements, and even uploading an image.
+
+I really liked this homework assignment and the overall simplicity of Markdown. The GitHub workflow was definitely pretty clunky and confusing at first but I think it will become more natural with more practice over time!
+
+---
```

Comment:
I like that you've added line dividers, it's a neat way of sectioning of the different segments